### PR TITLE
Invoke the correct lambda version.

### PR DESCRIPTION
**Pull request by AlvarHHM**
Changes to support the lambda versioning.

### CODE REVIEW

File: lambdawarmer/__init__.py [hunk 1]
```diff
@@ -105,7 +106,7 @@ def _perform_fan_out_warm_up_calls(config, correlation_id, concurrency, lambda_c
     for i in range(1, concurrency):
         try:
             lambda_client.invoke(
-                FunctionName=LAMBDA_INFO['function_name'],
+                FunctionName='%s:%s' % (LAMBDA_INFO['function_name'], LAMBDA_INFO['function_version']),
```
Comment (robhowley) [inline lambdawarmer/__init__.py]: please use the newer string format syntax. you could do ...
```
'{function_name}:{function_version}'.format(**LAMBDA_INFO)
```

**Comment by robhowley** [comment]
this is a great add, thanks @AlvarHHM. one small item and i'll give the 👍

File: lambdawarmer/__init__.py (hunk 1 above)
Comment (AlvarHHM) [reply lambdawarmer/__init__.py]: fixed

**Review by robhowley — APPROVED** [review]
i'll put it up in pypi later today

**Comment by robhowley** [comment]
@AlvarHHM feel free to merge whenever. it'll be avail soon enough afterward

**Comment by AlvarHHM** [comment]
> @AlvarHHM feel free to merge whenever. it'll be avail soon enough afterward

I am not super familiar with the pull request process, do I have to initiate the merge?

I thought the merge should be triggered on your end.

**Comment by robhowley** [comment]
just press the "Squash and merge" button. you earned it, you press it.

**Comment by AlvarHHM** [comment]
I am pretty sure I do not see the button and only the repo maintainer has the permission to merge. 

Screenshot 2019-08-15 at 2 09 51 PM


Perhaps use "rebase commit", if you want to preserve the author info. But I should be very happy as long as you can publish it on PyPI ASAP😁😁.
[https://community.atlassian.com/t5/Bitbucket-questions/How-to-preserve-the-pull-request-author-as-commit-author-in/qaq-p/892714](url)

**Comment by robhowley** [comment]
oh so sorry thought i had enabled that for approved prs

**Pull request merged**